clayui.com/src/templates/docs.js

```diff
@@ -176,14 +176,22 @@ export default (props) => {
 		location.hash === '#css' || !containsReact ? CSS_INDEX : REACT_INDEX
 	);
 
+	const [hasMounted, setHasMounted] = React.useState(false);
+
 	useEffect(() => {
+		setHasMounted(true);
+
```

Comment:
Ideally, a `useEffect` should deal with only one responsibility, it would be good to add it to another `useEffect` since it deals with something else.